docs/tempo/website/configuration/tls.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Configure TLS Communication
+weight: 70
+---
+
+# Configure TLS Communication
+
+Tempo can be configured to communicate between the components using Transport Layer Security, or TLS.
+
+Please note, the ciphers and TLS version here are for example purposes only. We are in no way recommending which ciphers or TLS versions for use in production environments, and are here for example purposes only.
+
+## Server Configuration
+
+```yaml
+server:
+  tls_cipher_suites: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
```

Comment:
is this just documenting weaveworks common config? fine if it is, just checking.

i assume this the list of accepted cipher suites?

Reply:
The specific list of ciphers isn't related to the weaveworks config beyond supporting the `tls_cipher_suites` config option.  Those were just ciphers that I had laying around for reasons, so just used those for examples.  Far more ciphers are supported, but selection is part art/science/paranoia, I suppose.